src/main/java/teammates/ui/pagedata/InstructorCourseEnrollPageData.java

```diff
@@ -1,17 +1,27 @@
 package teammates.ui.pagedata;
 
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
 
+import teammates.common.datatransfer.StudentUpdateStatus;
 import teammates.common.datatransfer.attributes.AccountAttributes;
+import teammates.common.datatransfer.attributes.StudentAttributes;
+import teammates.common.util.Logger;
 
 /**
  * PageData: this is page data for 'Enroll' page for a course of an instructor.
  */
 public class InstructorCourseEnrollPageData extends PageData {
+
+    private static final Logger log = Logger.getLogger();
+
     private String courseId;
     private String enrollStudents;
     private Map<String, String> enrollErrorLines = new HashMap<>();
+    private Map<String, String> enrollNewStudentsLines = new HashMap();
+    private Map<String, String> enrollModifiedStudentsLines = new HashMap();
+    private Map<String, String> enrollUnmodifiedStudentsLines = new HashMap();
```

Comment:
@_@ why so many `Map`...

Reply:
This is to cater to the different types of feedback message from the enrollment result page. Explained more in this https://github.com/TEAMMATES/teammates/pull/9017#discussion_r208454216.